dev/CommonStyles/ListViewItem_themeresources_21h1.xaml

```diff
@@ -843,21 +833,9 @@
                     <Border x:Name="OuterContainer" RenderTransformOrigin="0.5,0.5">
                         <VisualStateManager.VisualStateGroups>
                             <VisualStateGroup x:Name="CommonStates">
-                                <VisualState x:Name="Normal">
-                                    <Storyboard>
-                                        <PointerUpThemeAnimation Storyboard.TargetName="TiltContainer" />
```

Comment:
I also think the tilt containers can be removed as they're specifically there as wrappers for the animations

Reply:
I don't think we can @tashatitova, there might be someone using this (by manually extracting child) and it will break the code:)

Reply:
If someone is manually extracting children, that's a problem ;)  -  But really, if the only reason for this container is to power tilt, wouldn't it make sense to remove it? Yes, people might be doing that thing, but we should have ownership over the template.

Reply:
As @stmoy pointed out, WinUI has ownership of the template, if changes are made the template, usually, we don't care if someone digs into the template too deep since this would become very awkward for us and especially for the developers digging into a controls template from code behind. So if there is no reason for this to exist right now (beside it might break folks), you should be able to remove it. Otherwise, we might as well never change templates/template parts as we could break someone.

Reply:
Agree with @stmoy and @chingucoding There is extra perf cost to having extra elements in the tree, so if this is unused  now (and we don't expect it from code, which is very likely) lets remove it.

Reply:
@ranjeshj @stmoy @chingucoding @tashatitova I agree:) I'll address that in a separate PR